packages/truffle-core/lib/testing/testrunner.js

```diff
@@ -1,4 +1,5 @@
 var Web3 = require("web3");
+var LegacyWeb3 = require("legacy-web3");
```

Comment:
Here's an idea: what if, instead of my above proposed `truffle-contract-legacy`, we create a single new package `truffle-legacy-system` package, minimally containing things like `truffle-contract/lib/legacy.js`, along with exposing `Web3` from v0.20.6?

Then we could do:

```javascript
const Legacy = require("truffle-legacy-system");

this.web3 = options.legacy ? new Legacy.Web3() : new Web3();
```

And then we could avoid duplicating `web3` as `legacy-web3` just for package namespace reasons. `truffle-legacy-system` would have its own namespace, specifying the old Web3.js.